### PR TITLE
Add profile

### PR DESCRIPTION
**Pull request by Asura21**
Add profile.

### CODE REVIEW

File: content/authors/revanmuhammad/_index.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+- name: Sriwijaya University
+  url: "https://unsri.ac.id/"
+
+# Short bio (displayed in user profile at end of posts)
+bio: Student in Sriwijaya University
+
+interests:
+- Machine Learning
+- Software Engineering
+- Web Developer
```
Comment (ariefrahmansyah) [inline content/authors/revanmuhammad/_index.md]: It's better to write it as Web Development.

**Review by ariefrahmansyah — CHANGES_REQUESTED** [review]
Ditunggu foto nya ya

File: content/authors/revanmuhammad/_index.md (hunk 1 above)
Comment (datengyuk) [reply content/authors/revanmuhammad/_index.md]: Oke kak thanks for the correction

Comment (Asura21) [reply content/authors/revanmuhammad/_index.md]: Okeeeee kak, thanks for the correction

**Review by ariefrahmansyah — APPROVED** [review]

**Pull request merged**